ml-agents/mlagents/trainers/sac/optimizer_torch.py

```diff
@@ -115,24 +110,32 @@ def __init__(self, policy: TorchPolicy, trainer_params: TrainerSettings):
         )
         self.soft_update(self.policy.actor_critic.critic, self.target_network, 1.0)
 
+        _total_act_size = 0
```

Comment:
The ActionSpec (and so maybe the policy) has an `action_size` property to be backwards compatible with TF but also can be used for this.